src/components/FacebookButton.tsx

```diff
@@ -0,0 +1,23 @@
+import FacebookLogin from 'react-facebook-login';
+import { FACEBOOK_APP } from "../constants";
+
+const FacebookButton = () => {
+  const responseFacebook = (response: any) => {
+    console.log(response);
+  }
+
+  return (
+    <div>
+      <FacebookLogin
+        appId={FACEBOOK_APP.id}
+        fields="name,email,picture, user_birthday"
```

Comment:
Please write consistantly and set space after comma ',':

fields="name, email, picture, user_birthday"
scope="public_profile, user_friends, email, user_birthday"

Reply:
done

Reply:
You can not write 'done'. Just push 'Resolve conversation' button instead